pkgs/by-name/sr/srec/package.nix

```
@@ -0,0 +1,30 @@
+{ lib, clangStdenv, fetchFromGitHub }:
+let
+  name = "srec";
+  version = "0-unstable-2015-03-06";
+in
+clangStdenv.mkDerivation {
+
+  inherit name version;
```

```suggestion
{
  lib,
  clangStdenv,
  fetchFromGitHub,
}:
let
  pname = "srec";
  version = "0-unstable-2015-03-06";
in
clangStdenv.mkDerivation {
  inherit pname version;
```
package names should be `pname` instead of `name`, also formatted as per RFC 0166.